tests/unit-tests/models/runTest-test.js

```diff
@@ -151,7 +151,7 @@ describe('Run test', () => {
             metricsExportConfig: Buffer.from(JSON.stringify(consts.INFLUXDB_CONFIGURATION)).toString('base64')
         }
     ]
-    .forEach((testConfig) => {
+    .forEach((testConfig ) => {
```

Comment:
? did u mean to change here something

Reply:
Its just indentation I guess eslint didn't like it